### PR TITLE
XW-1873 | Normalize spacing on DS Global Footer for Mercury

### PR DESCRIPTION
**Pull request by kvas-damian**
## Links

* https://wikia-inc.atlassian.net/browse/XW-1873

## Description

Override mercury styles to make footer looks the same on all environments.

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: front/main/app/styles/component/_global-footer.scss [hunk 1]
```diff
@@ -4,16 +4,22 @@
...
 
-	&__home-of-fandom {
-		line-height: 1;
+	h2 {
+		line-height: normal;
+		font-size: 1.5em;
 	}
 }
 
+.wds-global-footer__section-header {
```
Comment (Warkot) [inline front/main/app/styles/component/_global-footer.scss]: I understand that `h2` inside the `.wds-global-footer {…}` is creating a stronger selector. But why do you need this one? And if you actually need it, you could write it `&__section-header` inside `.wds-global-footer {…}`.

**Comment by Grunny** [comment]
LGTM.

File: front/main/app/styles/component/_global-footer.scss [hunk 2]
```diff
@@ -4,14 +4,24 @@
...
 
 	.show-global-footer & {
 		display: block;
 	}
 
-	&__home-of-fandom {
-		line-height: 1;
+	h2 {
+		line-height: normal;
+		font-size: 1.5em;
```
Comment (Grunny) [inline front/main/app/styles/component/_global-footer.scss]: Minor: properties should be alphabetised per conventions.

File: front/main/app/styles/component/_global-footer.scss [hunk 3]
```diff
@@ -4,14 +4,24 @@
...
 	}
+
+	&__section-header {
+		line-height: normal;
+	}
+}
+
+.wds-global-footer__section-header {
```
Comment (Warkot) [inline front/main/app/styles/component/_global-footer.scss]: You might have forgot to remove this one after the last commit as it now duplicates it. ;)

**Comment by kvas-damian** [comment]
Sorry for that. I shouldn't submit this PR before the lunch ;)
Should be all fixed now.

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by Warkot** [comment]
If the values are in sync with mockups, then LGTM.

**Pull request merged**